stubs/README.md

```diff
@@ -0,0 +1,43 @@
+# AlgoPy - API for writing Algorand Python smart contracts
```

Comment:
```suggestion
# Algorand Python framework - API for writing Algorand Python smart contracts
```

Reply:
AlgoPy isn't the name of it, we don't want to have confusion by introducing extra terminology, `algopy` is simply the abbreviated import path for Algorand Python.

Reply:
unsure if we call it "framework" or "APIs"?

Reply:
It's both, but framework is the broader term so that gets my +1